sdks/java/io/google-cloud-platform/pom.xml

```diff
@@ -39,21 +39,13 @@
     <plugins>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
The groupId is not removed, it is still there, it uses the one of the previous line.